Description: `foo` is defined even when foo return non-zero
 (it is undefined only foo cannot be executed at all)
diff -dubr debhelper-8.9.13~bpo60+1/dh_installman debhelper-8.9.13~bpo60+1.man/dh_installman
--- debhelper-8.9.13~bpo60+1/dh_installman	2011-11-15 16:42:46.000000000 +0400
+++ debhelper-8.9.13~bpo60+1.man/dh_installman	2012-11-29 21:22:49.053102887 +0400
@@ -197,7 +197,7 @@
 	}
 
 	# Now utf-8 conversion.
-	if (defined `man --version`) {
+	if (0 == system('man --version >/dev/null 2>/dev/null')) {
 		foreach my $dir (qw{usr/share/man}) {
 			next unless -e "$tmp/$dir";
 			find(sub {
